app/api/bread/views.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+
+import flask_jwt_extended as jwt
+from flask import request
+from flask_restful import Resource
+
+from app import db
+from app.api import api
+from app.models import BreadOrder, BreadType
+from .schema import OrderSchema, BreadListSchema, BreadTypeSchema
+
+order_schema = OrderSchema(many=True)
+breadlist_schema = BreadListSchema()
+bread_types_schema = BreadTypeSchema(many=True)
+
+
+@api.resource('/bread/')
+class BreadOrderListResource(Resource):
+
+    @jwt.jwt_required
+    def get(self):
+        startdate = datetime.date.today()
+        if startdate.month >= 9:
+            startdate = startdate.replace(month=9, day=1)
+        else:
+            startdate = startdate.replace(month=1, day=1)
+
+        user = jwt.current_user
+        orders = BreadOrder.get_all_after_user(startdate, user)
+
+        for order in orders:
+            order.date = order.bread_date.date
```

Comment:
deze code maakt de view moeilijker leesbaar, ofwel:
1) zet dit in order_schema
2) doe de sql query beter zodat dat daar beter wordt teruggegeven